build.sbt

```diff
@@ -34,6 +34,37 @@ ThisBuild / printScalaVersion := {
   log.info(s"Local maven ${Resolver.mavenLocal}")
 }
 
+lazy val jacocoCoreReportSettings = Seq(
```

Comment:
Same feedback as in atum https://github.com/AbsaOSS/atum/pull/144#discussion_r1021678328

Reply:
Resolved by commit 58f921a73167354e524e63072311893ae5b23b94.